[{"id": 1613, "name": "22.txt", "raw": "2022-05-17 08:20:47,017 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********\n2022-05-17 08:20:47,017 | TASK [Wait for message 'Autoinstall is completed.' appear in VM log serial-20220517070407.log] \ntask path: /home/worker/workspace/Ansible_Cycle_Photon_3.x_Update/ansible-vsphere-gos-validation/common/vm_wait_log_msg.yml:35\nfatal: [localhost]: FAILED! => {\n    \"attempts\": 720,\n    \"censored\": \"the output has been hidden due to the fact that 'no_log: true' was specified for this result\",\n    \"changed\": false\n}\n2022-05-17 08:21:26,017 | TASK [Testing exit due to failure] *************************\ntask path: /home/worker/workspace/Ansible_Cycle_Photon_3.x_Update/ansible-vsphere-gos-validation/common/test_rescue.yml:55\nfatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_paravirtual_vmxnet3\n", "category": "output_hidden_no_log", "processed": "timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task wait for message auto install is completed appear in vm log serial timestamp log task path home worker workspace ansible cycle photon number x update ansible vsphere gos validation common vm wait log message yml number fatal localhost failed attempts number censored the output has been hidden due to the fact that no log true was specified for this result changed false timestamp task testing exit due to failure task path home worker workspace ansible cycle photon number x update ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi para virtual vmxnet number", "solution": "retry", "target": "targetvm", "version": 202206171000}, {"id": 1614, "name": "26.txt", "raw": "2022-05-16 15:14:19,016 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********\n2022-05-16 15:14:19,016 | TASK [Wait for message 'Autoinstall is completed.' appear in VM log serial-20220516135726.log] \ntask path: /home/worker/workspace/Ansible_CentOS_8.x_MAIN_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/vm_wait_log_msg.yml:35\nfatal: [localhost]: FAILED! => {\n    \"attempts\": 720,\n    \"censored\": \"the output has been hidden due to the fact that 'no_log: true' was specified for this result\",\n    \"changed\": false\n}\n2022-05-16 15:14:51,016 | TASK [Testing exit due to failure] *************************\ntask path: /home/worker/workspace/Ansible_CentOS_8.x_MAIN_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:55\nfatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_paravirtual_vmxnet3", "category": "output_hidden_no_log", "processed": "timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task wait for message auto install is completed appear in vm log serial timestamp log task path home worker workspace ansible centos number x main para virtual vmxnet number efi ansible vsphere gos validation common vm wait log message yml number fatal localhost failed attempts number censored the output has been hidden due to the fact that no log true was specified for this result changed false timestamp task testing exit due to failure task path home worker workspace ansible centos number x main para virtual vmxnet number efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi para virtual vmxnet number", "solution": "retry", "target": "targetvm", "version": 202206171000}, {"id": 1615, "name": "27.txt", "raw": "2022-05-17 05:42:48,017 | Failed at Play [deploy_vm_efi_nvme_vmxnet3] ****************\n2022-05-17 05:42:48,017 | TASK [Wait for message 'Autoinstall is completed.' appear in VM log serial-20220517042722.log] \ntask path: /home/worker/workspace/Ansible_RockyLinux_8.x_70U3_NVME_VMXNET3_EFI/ansible-vsphere-gos-validation/common/vm_wait_log_msg.yml:35\nfatal: [localhost]: FAILED! => {\n    \"attempts\": 720,\n    \"censored\": \"the output has been hidden due to the fact that 'no_log: true' was specified for this result\",\n    \"changed\": false\n}\n2022-05-17 05:43:21,017 | TASK [Testing exit due to failure] *************************\ntask path: /home/worker/workspace/Ansible_RockyLinux_8.x_70U3_NVME_VMXNET3_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:55\nfatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_nvme_vmxnet3", "category": "output_hidden_no_log", "processed": "timestamp failed at play deploy vm efi nvme vmxnet number timestamp task wait for message auto install is completed appear in vm log serial timestamp log task path home worker workspace ansible rocky linux number x nvme vmxnet number efi ansible vsphere gos validation common vm wait log message yml number fatal localhost failed attempts number censored the output has been hidden due to the fact that no log true was specified for this result changed false timestamp task testing exit due to failure task path home worker workspace ansible rocky linux number x nvme vmxnet number efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi nvme vmxnet number", "solution": "retry", "target": "targetvm", "version": 202206171000}, {"id": 1616, "name": "39.txt", "raw": "2022-05-17 06:01:20,017 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********\n2022-05-17 06:01:20,017 | TASK [Wait for message 'Autoinstall is completed.' appear in VM log serial-20220517044602.log] \ntask path: /home/worker/workspace/Ansible_RockyLinux_8.x_MAIN_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/vm_wait_log_msg.yml:35\nfatal: [localhost]: FAILED! => {\n    \"attempts\": 720,\n    \"censored\": \"the output has been hidden due to the fact that 'no_log: true' was specified for this result\",\n    \"changed\": false\n}\n2022-05-17 06:01:52,017 | TASK [Testing exit due to failure] *************************\ntask path: /home/worker/workspace/Ansible_RockyLinux_8.x_MAIN_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:55\nfatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_paravirtual_vmxnet3", "category": "output_hidden_no_log", "processed": "timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task wait for message auto install is completed appear in vm log serial timestamp log task path home worker workspace ansible rocky linux number x main para virtual vmxnet number efi ansible vsphere gos validation common vm wait log message yml number fatal localhost failed attempts number censored the output has been hidden due to the fact that no log true was specified for this result changed false timestamp task testing exit due to failure task path home worker workspace ansible rocky linux number x main para virtual vmxnet number efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi para virtual vmxnet number", "solution": "retry", "target": "targetvm", "version": 202206171000}, {"id": 1628, "name": "19.txt", "raw": "2022-06-06 08:39:53,006 | Failed at Play [vgauth_check_service] **********************\n2022-06-06 08:39:53,006 | TASK [Skip testcase: vgauth_check_service, reason: Blocked] \ntask path: /home/worker/workspace/Ansible_Windows_11_MAIN_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/skip_test_case.yml:20\nfatal: [localhost]: FAILED! => Test case 'vgauth_check_service' is blocked because VMware tools installed: False, running: False", "category": "vmtools_not_installed", "processed": "timestamp failed at play vg auth check service timestamp task skip test case vg auth check service reason blocked task path home worker workspace ansible windows number main para virtual vmxnet number efi ansible vsphere gos validation common skip test case yml number fatal localhost failed test case vg auth check service is blocked because vmware tools installed false running false", "solution": "deepdive", "target": "usererror", "version": 202206171000}, {"id": 1617, "name": "47.txt", "raw": "2022-05-17 19:06:08,017 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********\n2022-05-17 19:06:08,017 | TASK [Wait for message 'Autoinstall is completed.' appear in VM log serial-20220517175148.log] \ntask path: /home/worker/workspace/Ansible_Cycle_RHEL_9.x/ansible-vsphere-gos-validation/common/vm_wait_log_msg.yml:35\nfatal: [localhost]: FAILED! => {\n    \"attempts\": 720,\n    \"censored\": \"the output has been hidden due to the fact that 'no_log: true' was specified for this result\",\n    \"changed\": false\n}\n2022-05-17 19:06:41,017 | TASK [Testing exit due to failure] *************************\ntask path: /home/worker/workspace/Ansible_Cycle_RHEL_9.x/ansible-vsphere-gos-validation/common/test_rescue.yml:55\nfatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_paravirtual_vmxnet3", "category": "output_hidden_no_log", "processed": "timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task wait for message auto install is completed appear in vm log serial timestamp log task path home worker workspace ansible cycle rhel number x ansible vsphere gos validation common vm wait log message yml number fatal localhost failed attempts number censored the output has been hidden due to the fact that no log true was specified for this result changed false timestamp task testing exit due to failure task path home worker workspace ansible cycle rhel number x ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi para virtual vmxnet number", "solution": "retry", "target": "targetvm", "version": 202206171000}, {"id": 1618, "name": "48.txt", "raw": "2022-05-17 19:21:54,017 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********\n2022-05-17 19:21:54,017 | TASK [Wait for message 'Autoinstall is completed.' appear in VM log serial-20220517180421.log] \ntask path: /home/worker/workspace/Ansible_Cycle_Ubuntu_22.04_ISO/ansible-vsphere-gos-validation/common/vm_wait_log_msg.yml:35\nfatal: [localhost]: FAILED! => {\n    \"attempts\": 720,\n    \"censored\": \"the output has been hidden due to the fact that 'no_log: true' was specified for this result\",\n    \"changed\": false\n}\n2022-05-17 19:22:28,017 | TASK [Testing exit due to failure] *************************\ntask path: /home/worker/workspace/Ansible_Cycle_Ubuntu_22.04_ISO/ansible-vsphere-gos-validation/common/test_rescue.yml:55\nfatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_paravirtual_vmxnet3", "category": "output_hidden_no_log", "processed": "timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task wait for message auto install is completed appear in vm log serial timestamp log task path home worker workspace ansible cycle ubuntu number iso ansible vsphere gos validation common vm wait log message yml number fatal localhost failed attempts number censored the output has been hidden due to the fact that no log true was specified for this result changed false timestamp task testing exit due to failure task path home worker workspace ansible cycle ubuntu number iso ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi para virtual vmxnet number", "solution": "retry", "target": "targetvm", "version": 202206171000}, {"id": 1619, "name": "80.txt", "raw": "2022-05-30 07:33:38,030 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********\n2022-05-30 07:33:38,030 | TASK [Wait for message 'Autoinstall is completed.' appear in VM log serial-20220530054052.log] \ntask path: /home/worker/workspace/Ansible_RHEL_9.x_MAIN_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/vm_wait_log_msg.yml:35\nfatal: [localhost]: FAILED! => {\n    \"attempts\": 720,\n    \"censored\": \"the output has been hidden due to the fact that 'no_log: true' was specified for this result\",\n    \"changed\": false\n}\n2022-05-30 07:34:10,030 | TASK [Testing exit due to failure] *************************\ntask path: /home/worker/workspace/Ansible_RHEL_9.x_MAIN_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:55\nfatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_paravirtual_vmxnet3", "category": "output_hidden_no_log", "processed": "timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task wait for message auto install is completed appear in vm log serial timestamp log task path home worker workspace ansible rhel number x main para virtual vmxnet number efi ansible vsphere gos validation common vm wait log message yml number fatal localhost failed attempts number censored the output has been hidden due to the fact that no log true was specified for this result changed false timestamp task testing exit due to failure task path home worker workspace ansible rhel number x main para virtual vmxnet number efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi para virtual vmxnet number", "solution": "retry", "target": "targetvm", "version": 202206171000}, {"id": 1625, "name": "1.txt", "raw": "2022-06-06 07:38:54,006 | Failed at Play [mouse_driver_vmtools] **********************\n2022-06-06 07:38:54,006 | TASK [Skip testcase: mouse_driver_vmtools, reason: Blocked] \ntask path: /home/worker/workspace/Ansible_Windows_Server_LTSC_MAIN_PARAVIRTUAL_VMXNET3_BIOS/ansible-vsphere-gos-validation/common/skip_test_case.yml:20\nfatal: [localhost]: FAILED! => Test case 'mouse_driver_vmtools' is blocked because VMware tools installed: False, running: False", "category": "vmtools_not_installed", "processed": "timestamp failed at play mouse driver vm tools timestamp task skip test case mouse driver vm tools reason blocked task path home worker workspace ansible windows server lts c main para virtual vmxnet number bios ansible vsphere gos validation common skip test case yml number fatal localhost failed test case mouse driver vm tools is blocked because vmware tools installed false running false", "solution": "deepdive", "target": "usererror", "version": 202206171000}, {"id": 1620, "name": "89.txt", "raw": "2022-06-03 00:50:07,003 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********\n2022-06-03 00:50:07,003 | TASK [Wait for message 'Autoinstall is completed.' appear in VM log serial-20220602233434.log] \ntask path: /home/worker/workspace/Ansible_Cycle_CentOS_8.x/ansible-vsphere-gos-validation/common/vm_wait_log_msg.yml:35\nfatal: [localhost]: FAILED! => {\n    \"attempts\": 720,\n    \"censored\": \"the output has been hidden due to the fact that 'no_log: true' was specified for this result\",\n    \"changed\": false\n}\n2022-06-03 00:50:42,003 | TASK [Testing exit due to failure] *************************\ntask path: /home/worker/workspace/Ansible_Cycle_CentOS_8.x/ansible-vsphere-gos-validation/common/test_rescue.yml:55\nfatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_paravirtual_vmxnet3", "category": "output_hidden_no_log", "processed": "timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task wait for message auto install is completed appear in vm log serial timestamp log task path home worker workspace ansible cycle centos number x ansible vsphere gos validation common vm wait log message yml number fatal localhost failed attempts number censored the output has been hidden due to the fact that no log true was specified for this result changed false timestamp task testing exit due to failure task path home worker workspace ansible cycle centos number x ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi para virtual vmxnet number", "solution": "retry", "target": "targetvm", "version": 202206171000}, {"id": 1621, "name": "58.txt", "raw": "2022-05-23 16:42:16,023 | Failed at Play [lsilogic_vhba_device_ops] ******************\n2022-05-23 16:42:16,023 | TASK [Wait for new device to be present] *******************\ntask path: /home/worker/workspace/Ansible_Regression_Ubuntu_22.04_Server_ISO/ansible-vsphere-gos-validation/linux/vhba_hot_add_remove/wait_device_list_changed.yml:80\nfatal: [localhost]: UNREACHABLE! => Failed to create temporary directory. In some cases, you may have been able to authenticate and did not have permissions on the target directory. Consider changing the remote tmp path in ansible.cfg to a path rooted in \"/tmp\", for more error information use -vvv. Failed command was: ( umask 77 && mkdir -p \"` echo /tmp `\"&& mkdir \"` echo /tmp/ansible-tmp-1653324136.7846885-16090-258133457335772 `\" && echo ansible-tmp-1653324136.7846885-16090-258133457335772=\"` echo /tmp/ansible-tmp-1653324136.7846885-16090-258133457335772 `\" ), exited with result 1\n2022-05-23 16:42:16,023 | TASK [Guest OS unreachable] ********************************\ntask path: /home/worker/workspace/Ansible_Regression_Ubuntu_22.04_Server_ISO/ansible-vsphere-gos-validation/linux/vhba_hot_add_remove/wait_device_list_changed.yml:91\nfatal: [localhost]: FAILED! => {'changed': False, 'msg': 'Failed to create temporary directory. In some cases, you may have been able to authenticate and did not have permissions on the target directory. Consider changing the remote tmp path in ansible.cfg to a path rooted in \"/tmp\", for more error information use -vvv. Failed command was: ( umask 77 && mkdir -p \"` echo /tmp `\"&& mkdir \"` echo /tmp/ansible-tmp-1653324136.7846885-16090-258133457335772 `\" && echo ansible-tmp-1653324136.7846885-16090-258133457335772=\"` echo /tmp/ansible-tmp-1653324136.7846885-16090-258133457335772 `\" ), exited with result 1', 'skip_reason': 'Host localhost is unreachable', 'unreachable': True}", "category": "fail_to_create_tem_dir", "processed": "timestamp failed at play lsi logic v hba device ops timestamp task wait for new device to be present task path home worker workspace ansible regression ubuntu number server iso ansible vsphere gos validation linux v hba hot add remove wait device list changed yml number fatal localhost un reachable failed to create temporary directory in some cases you may have been able to authenticate and did not have permissions on the target directory consider changing the remote tmp path in ansible configuration to a path rooted in tmp for more error information use v v v failed command was u mask number mkdir p echo tmp mkdir echo tmp ansible tmp hex id number timestamp echo ansible tmp hex id number timestamp echo tmp ansible tmp hex id number timestamp exited with result number timestamp task guest os un reachable task path home worker workspace ansible regression ubuntu number server iso ansible vsphere gos validation linux v hba hot add remove wait device list changed yml number fatal localhost failed changed false message failed to create temporary directory in some cases you may have been able to authenticate and did not have permissions on the target directory consider changing the remote tmp path in ansible configuration to a path rooted in tmp for more error information use v v v failed command was u mask number mkdir p echo tmp mkdir echo tmp ansible tmp hex id number timestamp echo ansible tmp hex id number timestamp echo tmp ansible tmp hex id number timestamp exited with result number skip reason host localhost is un reachable un reachable true", "solution": "deepdive", "target": "testcase", "version": 202206171000}, {"id": 1624, "name": "0.txt", "raw": "2022-06-06 06:03:18,006 | Failed at Play [mouse_driver_vmtools] **********************\n2022-06-06 06:03:18,006 | TASK [Skip testcase: mouse_driver_vmtools, reason: Blocked] \ntask path: /home/worker/workspace/Ansible_Windows_Server_LTSC_MAIN_NVME_E1000E_BIOS/ansible-vsphere-gos-validation/common/skip_test_case.yml:20\nfatal: [localhost]: FAILED! => Test case 'mouse_driver_vmtools' is blocked because VMware tools installed: False, running: False", "category": "vmtools_not_installed", "processed": "timestamp failed at play mouse driver vm tools timestamp task skip test case mouse driver vm tools reason blocked task path home worker workspace ansible windows server lts c main nvme bios ansible vsphere gos validation common skip test case yml number fatal localhost failed test case mouse driver vm tools is blocked because vmware tools installed false running false", "solution": "deepdive", "target": "usererror", "version": 202206171000}, {"id": 1626, "name": "17.txt", "raw": "2022-06-06 08:39:01,006 | Failed at Play [vgauth_check_service] **********************\n2022-06-06 08:39:01,006 | TASK [Skip testcase: vgauth_check_service, reason: Blocked] \ntask path: /home/worker/workspace/Ansible_Windows_11_MAIN_NVME_E1000E_EFI/ansible-vsphere-gos-validation/common/skip_test_case.yml:20\nfatal: [localhost]: FAILED! => Test case 'vgauth_check_service' is blocked because VMware tools installed: False, running: False", "category": "vmtools_not_installed", "processed": "timestamp failed at play vg auth check service timestamp task skip test case vg auth check service reason blocked task path home worker workspace ansible windows number main nvme efi ansible vsphere gos validation common skip test case yml number fatal localhost failed test case vg auth check service is blocked because vmware tools installed false running false", "solution": "deepdive", "target": "usererror", "version": 202206171000}, {"id": 1627, "name": "18.txt", "raw": "2022-06-06 07:44:14,006 | Failed at Play [vgauth_check_service] **********************\n2022-06-06 07:44:14,006 | TASK [Skip testcase: vgauth_check_service, reason: Blocked] \ntask path: /home/worker/workspace/Ansible_Windows_Server_LTSC_MAIN_NVME_E1000E_BIOS/ansible-vsphere-gos-validation/common/skip_test_case.yml:20\nfatal: [localhost]: FAILED! => Test case 'vgauth_check_service' is blocked because VMware tools installed: False, running: False", "category": "vmtools_not_installed", "processed": "timestamp failed at play vg auth check service timestamp task skip test case vg auth check service reason blocked task path home worker workspace ansible windows server lts c main nvme bios ansible vsphere gos validation common skip test case yml number fatal localhost failed test case vg auth check service is blocked because vmware tools installed false running false", "solution": "deepdive", "target": "usererror", "version": 202206171000}, {"id": 1629, "name": "2.txt", "raw": "2022-06-06 06:22:35,006 | Failed at Play [mouse_driver_vmtools] **********************\n2022-06-06 06:22:35,006 | TASK [Skip testcase: mouse_driver_vmtools, reason: Blocked] \ntask path: /home/worker/workspace/Ansible_Windows_11_MAIN_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/skip_test_case.yml:20\nfatal: [localhost]: FAILED! => Test case 'mouse_driver_vmtools' is blocked because VMware tools installed: False, running: False", "category": "vmtools_not_installed", "processed": "timestamp failed at play mouse driver vm tools timestamp task skip test case mouse driver vm tools reason blocked task path home worker workspace ansible windows number main para virtual vmxnet number efi ansible vsphere gos validation common skip test case yml number fatal localhost failed test case mouse driver vm tools is blocked because vmware tools installed false running false", "solution": "deepdive", "target": "usererror", "version": 202206171000}, {"id": 1630, "name": "3.txt", "raw": "2022-06-06 07:38:49,006 | Failed at Play [mouse_driver_vmtools] **********************\n2022-06-06 07:38:49,006 | TASK [Skip testcase: mouse_driver_vmtools, reason: Blocked] \ntask path: /home/worker/workspace/Ansible_Windows_Server_LTSC_MAIN_LSILOGICSAS_E1000E_EFI/ansible-vsphere-gos-validation/common/skip_test_case.yml:20\nfatal: [localhost]: FAILED! => Test case 'mouse_driver_vmtools' is blocked because VMware tools installed: False, running: False", "category": "vmtools_not_installed", "processed": "timestamp failed at play mouse driver vm tools timestamp task skip test case mouse driver vm tools reason blocked task path home worker workspace ansible windows server lts c main lsi logic sas efi ansible vsphere gos validation common skip test case yml number fatal localhost failed test case mouse driver vm tools is blocked because vmware tools installed false running false", "solution": "deepdive", "target": "usererror", "version": 202206171000}, {"id": 1631, "name": "4.txt", "raw": "2022-06-06 08:37:33,006 | Failed at Play [mouse_driver_vmtools] **********************\n2022-06-06 08:37:33,006 | TASK [Skip testcase: mouse_driver_vmtools, reason: Blocked] \ntask path: /home/worker/workspace/Ansible_Windows_11_MAIN_NVME_E1000E_EFI/ansible-vsphere-gos-validation/common/skip_test_case.yml:20\nfatal: [localhost]: FAILED! => Test case 'mouse_driver_vmtools' is blocked because VMware tools installed: False, running: False", "category": "vmtools_not_installed", "processed": "timestamp failed at play mouse driver vm tools timestamp task skip test case mouse driver vm tools reason blocked task path home worker workspace ansible windows number main nvme efi ansible vsphere gos validation common skip test case yml number fatal localhost failed test case mouse driver vm tools is blocked because vmware tools installed false running false", "solution": "deepdive", "target": "usererror", "version": 202206171000}, {"id": 1632, "name": "5.txt", "raw": "2022-06-06 08:41:44,006 | Failed at Play [stat_balloon] ******************************\n2022-06-06 08:41:44,006 | TASK [Skip testcase: stat_balloon, reason: Blocked] ********\ntask path: /home/worker/workspace/Ansible_Windows_11_MAIN_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/skip_test_case.yml:20\nfatal: [localhost]: FAILED! => Test case 'stat_balloon' is blocked because VMware tools installed: False, running: False", "category": "vmtools_not_installed", "processed": "timestamp failed at play stat balloon timestamp task skip test case stat balloon reason blocked task path home worker workspace ansible windows number main para virtual vmxnet number efi ansible vsphere gos validation common skip test case yml number fatal localhost failed test case stat balloon is blocked because vmware tools installed false running false", "solution": "deepdive", "target": "usererror", "version": 202206171000}, {"id": 1633, "name": "6.txt", "raw": "2022-06-06 07:42:00,006 | Failed at Play [stat_balloon] ******************************\n2022-06-06 07:42:00,006 | TASK [Skip testcase: stat_balloon, reason: Blocked] ********\ntask path: /home/worker/workspace/Ansible_Windows_Server_LTSC_MAIN_PARAVIRTUAL_VMXNET3_BIOS/ansible-vsphere-gos-validation/common/skip_test_case.yml:20\nfatal: [localhost]: FAILED! => Test case 'stat_balloon' is blocked because VMware tools installed: False, running: False", "category": "vmtools_not_installed", "processed": "timestamp failed at play stat balloon timestamp task skip test case stat balloon reason blocked task path home worker workspace ansible windows server lts c main para virtual vmxnet number bios ansible vsphere gos validation common skip test case yml number fatal localhost failed test case stat balloon is blocked because vmware tools installed false running false", "solution": "deepdive", "target": "usererror", "version": 202206171000}, {"id": 1683, "name": "10.txt", "raw": "2022-05-25 16:49:06,025 | Failed at Play [testbed_deploy_nimbus] *********************\n2022-05-25 16:49:06,025 | TASK [Testbed deploy result is not PASS] *******************\ntask path: /home/worker/workspace/Ansible_SLED_15SP4_MAIN_NVME_E1000E_BIOS/newgos_testing_internal/testbed_deploy/parse_testbed_info.yml:10\nfatal: [localhost]: FAILED! => Get deployment result in testbedInfo.json is 'FAIL', not PASS", "category": "testbedinfo_fail", "processed": "timestamp failed at play testbed deploy nimbus timestamp task testbed deploy result is not pass task path home worker workspace ansible sled main nvme bios new gos testing internal testbed deploy parse testbed info yml number fatal localhost failed get deployment result in testbed info json is fail not pass", "solution": "retry", "target": "nimbus", "version": 202206171000}, {"id": 1634, "name": "7.txt", "raw": "2022-06-06 06:06:32,006 | Failed at Play [stat_balloon] ******************************\n2022-06-06 06:06:32,006 | TASK [Skip testcase: stat_balloon, reason: Blocked] ********\ntask path: /home/worker/workspace/Ansible_Windows_Server_LTSC_MAIN_NVME_E1000E_BIOS/ansible-vsphere-gos-validation/common/skip_test_case.yml:20\nfatal: [localhost]: FAILED! => Test case 'stat_balloon' is blocked because VMware tools installed: False, running: False", "category": "vmtools_not_installed", "processed": "timestamp failed at play stat balloon timestamp task skip test case stat balloon reason blocked task path home worker workspace ansible windows server lts c main nvme bios ansible vsphere gos validation common skip test case yml number fatal localhost failed test case stat balloon is blocked because vmware tools installed false running false", "solution": "deepdive", "target": "usererror", "version": 202206171000}, {"id": 1635, "name": "70.txt", "raw": "2022-05-25 14:40:06,025 | Failed at Play [stat_balloon] ******************************\n2022-05-25 14:40:06,025 | TASK [Skip testcase: stat_balloon, reason: Blocked] ********\ntask path: /home/worker/workspace/Ansible_Cycle_Windows_10_64/ansible-vsphere-gos-validation/common/skip_test_case.yml:20\nfatal: [localhost]: FAILED! => Test case 'stat_balloon' is blocked because VMware tools installed: False, running: False", "category": "vmtools_not_installed", "processed": "timestamp failed at play stat balloon timestamp task skip test case stat balloon reason blocked task path home worker workspace ansible cycle windows number ansible vsphere gos validation common skip test case yml number fatal localhost failed test case stat balloon is blocked because vmware tools installed false running false", "solution": "deepdive", "target": "usererror", "version": 202206171000}, {"id": 1636, "name": "8.txt", "raw": "2022-06-06 07:40:32,006 | Failed at Play [stat_balloon] ******************************\n2022-06-06 07:40:32,006 | TASK [Skip testcase: stat_balloon, reason: Blocked] ********\ntask path: /home/worker/workspace/Ansible_Windows_Server_LTSC_MAIN_LSILOGICSAS_E1000E_EFI/ansible-vsphere-gos-validation/common/skip_test_case.yml:20\nfatal: [localhost]: FAILED! => Test case 'stat_balloon' is blocked because VMware tools installed: False, running: False", "category": "vmtools_not_installed", "processed": "timestamp failed at play stat balloon timestamp task skip test case stat balloon reason blocked task path home worker workspace ansible windows server lts c main lsi logic sas efi ansible vsphere gos validation common skip test case yml number fatal localhost failed test case stat balloon is blocked because vmware tools installed false running false", "solution": "deepdive", "target": "usererror", "version": 202206171000}, {"id": 1637, "name": "9.txt", "raw": "2022-06-06 08:40:53,006 | Failed at Play [stat_balloon] ******************************\n2022-06-06 08:40:53,006 | TASK [Skip testcase: stat_balloon, reason: Blocked] ********\ntask path: /home/worker/workspace/Ansible_Windows_11_MAIN_NVME_E1000E_EFI/ansible-vsphere-gos-validation/common/skip_test_case.yml:20\nfatal: [localhost]: FAILED! => Test case 'stat_balloon' is blocked because VMware tools installed: False, running: False", "category": "vmtools_not_installed", "processed": "timestamp failed at play stat balloon timestamp task skip test case stat balloon reason blocked task path home worker workspace ansible windows number main nvme efi ansible vsphere gos validation common skip test case yml number fatal localhost failed test case stat balloon is blocked because vmware tools installed false running false", "solution": "deepdive", "target": "usererror", "version": 202206171000}, {"id": 1638, "name": "92.txt", "raw": "2022-06-06 07:40:59,006 | Failed at Play [vgauth_check_service] **********************\n2022-06-06 07:40:59,006 | TASK [Skip testcase: vgauth_check_service, reason: Blocked] \ntask path: /home/worker/workspace/Ansible_Windows_Server_LTSC_MAIN_PARAVIRTUAL_VMXNET3_BIOS/ansible-vsphere-gos-validation/common/skip_test_case.yml:20\nfatal: [localhost]: FAILED! => Test case 'vgauth_check_service' is blocked because VMware tools installed: False, running: False", "category": "vmtools_not_installed", "processed": "timestamp failed at play vg auth check service timestamp task skip test case vg auth check service reason blocked task path home worker workspace ansible windows server lts c main para virtual vmxnet number bios ansible vsphere gos validation common skip test case yml number fatal localhost failed test case vg auth check service is blocked because vmware tools installed false running false", "solution": "deepdive", "target": "usererror", "version": 202206171000}, {"id": 1639, "name": "11.txt", "raw": "2022-05-17 16:43:54,017 | Failed at Play [testbed_deploy_nimbus] *********************\n2022-05-17 16:43:54,017 | TASK [fail] ************************************************\ntask path: /home/worker/workspace/Ansible_Cycle_AlmaLinux_9.x/newgos_testing_internal/testbed_deploy/deploy_from_dbc_run.yml:75\nfatal: [localhost]: FAILED! => Failed to fetch nimbus testbed deploy result http://sc-dbc2154.eng.vmware.com/qiz/nimbus/Ansible_Cycle_AlmaLinux_9.x-1/testbed_result/testbedInfo.json from dbc server sc-dbc2154.eng.vmware.com. Nimbus testbed deployment failed.", "category": "fail_fetch_nimbus_result", "processed": "timestamp failed at play testbed deploy nimbus timestamp task fail task path home worker workspace ansible cycle alma linux number x new gos testing internal testbed deploy deploy from database c run yml number fatal localhost failed failed to fetch nimbus testbed deploy result http sc eng vmware com qiz nimbus ansible cycle alma linux number x number testbed result testbed info json from database c servers c eng vmware com nimbus testbed deployment failed", "solution": "retry", "target": "nimbus", "version": 202206171000}, {"id": 1640, "name": "12.txt", "raw": "2022-05-17 16:45:07,017 | Failed at Play [testbed_deploy_nimbus] *********************\n2022-05-17 16:45:07,017 | TASK [fail] ************************************************\ntask path: /home/worker/workspace/Ansible_Cycle_RockyLinux_8.x/newgos_testing_internal/testbed_deploy/deploy_from_dbc_run.yml:75\nfatal: [localhost]: FAILED! => Failed to fetch nimbus testbed deploy result http://sc-dbc2154.eng.vmware.com/qiz/nimbus/Ansible_Cycle_RockyLinux_8.x-88/testbed_result/testbedInfo.json from dbc server sc-dbc2154.eng.vmware.com. Nimbus testbed deployment failed.", "category": "fail_fetch_nimbus_result", "processed": "timestamp failed at play testbed deploy nimbus timestamp task fail task path home worker workspace ansible cycle rocky linux number x new gos testing internal testbed deploy deploy from database c run yml number fatal localhost failed failed to fetch nimbus testbed deploy result http sc eng vmware com qiz nimbus ansible cycle rocky linux number x number testbed result testbed info json from database c servers c eng vmware com nimbus testbed deployment failed", "solution": "retry", "target": "nimbus", "version": 202206171000}, {"id": 1641, "name": "13.txt", "raw": "2022-05-17 17:38:03,017 | Failed at Play [testbed_deploy_nimbus] *********************\n2022-05-17 17:38:03,017 | TASK [fail] ************************************************\ntask path: /home/worker/workspace/Ansible_Cycle_Windows_11_64/newgos_testing_internal/testbed_deploy/deploy_from_dbc_run.yml:75\nfatal: [localhost]: FAILED! => Failed to fetch nimbus testbed deploy result http://sc-dbc2154.eng.vmware.com/qiz/nimbus/Ansible_Cycle_Windows_11_64-119/testbed_result/testbedInfo.json from dbc server sc-dbc2154.eng.vmware.com. Nimbus testbed deployment failed.", "category": "fail_fetch_nimbus_result", "processed": "timestamp failed at play testbed deploy nimbus timestamp task fail task path home worker workspace ansible cycle windows number new gos testing internal testbed deploy deploy from database c run yml number fatal localhost failed failed to fetch nimbus testbed deploy result http sc eng vmware com qiz nimbus ansible cycle windows number testbed result testbed info json from database c servers c eng vmware com nimbus testbed deployment failed", "solution": "retry", "target": "nimbus", "version": 202206171000}, {"id": 1642, "name": "14.txt", "raw": "2022-05-17 19:23:08,017 | Failed at Play [testbed_deploy_nimbus] *********************\n2022-05-17 19:23:08,017 | TASK [fail] ************************************************\ntask path: /home/worker/workspace/Ansible_Cycle_SLED_15.x/newgos_testing_internal/testbed_deploy/deploy_from_dbc_run.yml:75\nfatal: [localhost]: FAILED! => Failed to fetch nimbus testbed deploy result http://sc-dbc2154.eng.vmware.com/qiz/nimbus/Ansible_Cycle_SLED_15.x-82/testbed_result/testbedInfo.json from dbc server sc-dbc2154.eng.vmware.com. Nimbus testbed deployment failed.", "category": "fail_fetch_nimbus_result", "processed": "timestamp failed at play testbed deploy nimbus timestamp task fail task path home worker workspace ansible cycle sled number x new gos testing internal testbed deploy deploy from database c run yml number fatal localhost failed failed to fetch nimbus testbed deploy result http sc eng vmware com qiz nimbus ansible cycle sled number x number testbed result testbed info json from database c servers c eng vmware com nimbus testbed deployment failed", "solution": "retry", "target": "nimbus", "version": 202206171000}, {"id": 1643, "name": "54.txt", "raw": "2022-05-16 13:43:47,016 | Failed at Play [testbed_deploy_nimbus] *********************\n2022-05-16 13:43:47,016 | TASK [fail] ************************************************\ntask path: /home/worker/workspace/Ansible_Flatcar_main/newgos_testing_internal/testbed_deploy/deploy_from_dbc_run.yml:75\nfatal: [localhost]: FAILED! => Failed to fetch nimbus testbed deploy result http://sc-dbc2154.eng.vmware.com/qiz/nimbus/Ansible_Flatcar_main-39/testbed_result/testbedInfo.json from dbc server sc-dbc2154.eng.vmware.com. Nimbus testbed deployment failed.", "category": "fail_fetch_nimbus_result", "processed": "timestamp failed at play testbed deploy nimbus timestamp task fail task path home worker workspace ansible flatcar main new gos testing internal testbed deploy deploy from database c run yml number fatal localhost failed failed to fetch nimbus testbed deploy result http sc eng vmware com qiz nimbus ansible flatcar main number testbed result testbed info json from database c servers c eng vmware com nimbus testbed deployment failed", "solution": "retry", "target": "nimbus", "version": 202206171000}, {"id": 1686, "name": "50.txt", "raw": "2022-05-16 08:04:47,016 | Failed at Play [testbed_deploy_nimbus] *********************\n2022-05-16 08:04:47,016 | TASK [Testbed deploy result is not PASS] *******************\ntask path: /home/worker/workspace/Ansible_Flatcar_70U3/newgos_testing_internal/testbed_deploy/parse_testbed_info.yml:10\nfatal: [localhost]: FAILED! => Get deployment result in testbedInfo.json is 'INVALID', not PASS", "category": "testbedinfo_fail", "processed": "timestamp failed at play testbed deploy nimbus timestamp task testbed deploy result is not pass task path home worker workspace ansible flatcar new gos testing internal testbed deploy parse testbed info yml number fatal localhost failed get deployment result in testbed info json is invalid not pass", "solution": "retry", "target": "nimbus", "version": 202206171000}, {"id": 1644, "name": "64.txt", "raw": "2022-05-24 08:22:04,024 | Failed at Play [testbed_deploy_nimbus] *********************\n2022-05-24 08:22:04,024 | TASK [fail] ************************************************\ntask path: /home/worker/workspace/Ansible_Cycle_RHEL_7.x/newgos_testing_internal/testbed_deploy/deploy_from_dbc_run.yml:75\nfatal: [localhost]: FAILED! => Failed to fetch nimbus testbed deploy result http://sc-dbc2154.eng.vmware.com/qiz/nimbus/Ansible_Cycle_RHEL_7.x-85/testbed_result/testbedInfo.json from dbc server sc-dbc2154.eng.vmware.com. Nimbus testbed deployment failed.", "category": "fail_fetch_nimbus_result", "processed": "timestamp failed at play testbed deploy nimbus timestamp task fail task path home worker workspace ansible cycle rhel number x new gos testing internal testbed deploy deploy from database c run yml number fatal localhost failed failed to fetch nimbus testbed deploy result http sc eng vmware com qiz nimbus ansible cycle rhel number x number testbed result testbed info json from database c servers c eng vmware com nimbus testbed deployment failed", "solution": "retry", "target": "nimbus", "version": 202206171000}, {"id": 1645, "name": "65.txt", "raw": "2022-05-17 16:43:37,017 | Failed at Play [testbed_deploy_nimbus] *********************\n2022-05-17 16:43:37,017 | TASK [fail] ************************************************\ntask path: /home/worker/workspace/Ansible_Cycle_OracleLinux_8.x/newgos_testing_internal/testbed_deploy/deploy_from_dbc_run.yml:75\nfatal: [localhost]: FAILED! => Failed to fetch nimbus testbed deploy result http://sc-dbc2154.eng.vmware.com/qiz/nimbus/Ansible_Cycle_OracleLinux_8.x-77/testbed_result/testbedInfo.json from dbc server sc-dbc2154.eng.vmware.com. Nimbus testbed deployment failed.", "category": "fail_fetch_nimbus_result", "processed": "timestamp failed at play testbed deploy nimbus timestamp task fail task path home worker workspace ansible cycle oracle linux number x new gos testing internal testbed deploy deploy from database c run yml number fatal localhost failed failed to fetch nimbus testbed deploy result http sc eng vmware com qiz nimbus ansible cycle oracle linux number x number testbed result testbed info json from database c servers c eng vmware com nimbus testbed deployment failed", "solution": "retry", "target": "nimbus", "version": 202206171000}, {"id": 1646, "name": "67.txt", "raw": "2022-05-25 05:40:55,025 | Failed at Play [testbed_deploy_nimbus] *********************\n2022-05-25 05:40:55,025 | TASK [fail] ************************************************\ntask path: /home/worker/workspace/Ansible_Windows_Server_LTSC_70U1_IDE_E1000E_BIOS/newgos_testing_internal/testbed_deploy/deploy_from_dbc_run.yml:75\nfatal: [localhost]: FAILED! => Failed to fetch nimbus testbed deploy result http://sc-dbc2154.eng.vmware.com/qiz/nimbus/Ansible_Windows_Server_LTSC_70U1_IDE_E1000E_BIOS-32/testbed_result/testbedInfo.json from dbc server sc-dbc2154.eng.vmware.com. Nimbus testbed deployment failed.", "category": "fail_fetch_nimbus_result", "processed": "timestamp failed at play testbed deploy nimbus timestamp task fail task path home worker workspace ansible windows server lts c ide bios new gos testing internal testbed deploy deploy from database c run yml number fatal localhost failed failed to fetch nimbus testbed deploy result http sc eng vmware com qiz nimbus ansible windows server lts c ide bios number testbed result testbed info json from database c servers c eng vmware com nimbus testbed deployment failed", "solution": "retry", "target": "nimbus", "version": 202206171000}, {"id": 1647, "name": "74.txt", "raw": "2022-05-26 05:43:00,026 | Failed at Play [testbed_deploy_nimbus] *********************\n2022-05-26 05:43:00,026 | TASK [fail] ************************************************\ntask path: /home/worker/workspace/Ansible_Windows_Server_LTSC_70U1_SATA_E1000E_EFI/newgos_testing_internal/testbed_deploy/deploy_from_dbc_run.yml:75\nfatal: [localhost]: FAILED! => Failed to fetch nimbus testbed deploy result http://sc-dbc2154.eng.vmware.com/qiz/nimbus/Ansible_Windows_Server_LTSC_70U1_SATA_E1000E_EFI-31/testbed_result/testbedInfo.json from dbc server sc-dbc2154.eng.vmware.com. Nimbus testbed deployment failed.", "category": "fail_fetch_nimbus_result", "processed": "timestamp failed at play testbed deploy nimbus timestamp task fail task path home worker workspace ansible windows server lts c sata efi new gos testing internal testbed deploy deploy from database c run yml number fatal localhost failed failed to fetch nimbus testbed deploy result http sc eng vmware com qiz nimbus ansible windows server lts c sata efi number testbed result testbed info json from database c servers c eng vmware com nimbus testbed deployment failed", "solution": "retry", "target": "nimbus", "version": 202206171000}, {"id": 1648, "name": "77.txt", "raw": "2022-05-26 17:55:18,026 | Failed at Play [testbed_deploy_nimbus] *********************\n2022-05-26 17:55:18,026 | TASK [fail] ************************************************\ntask path: /home/worker/workspace/Ansible_AlmaLinux_9.x_70U3_NVME_VMXNET3_EFI/newgos_testing_internal/testbed_deploy/deploy_from_dbc_run.yml:75\nfatal: [localhost]: FAILED! => Failed to fetch nimbus testbed deploy result http://sc-dbc2154.eng.vmware.com/qiz/nimbus/Ansible_AlmaLinux_9.x_70U3_NVME_VMXNET3_EFI-18/testbed_result/testbedInfo.json from dbc server sc-dbc2154.eng.vmware.com. Nimbus testbed deployment failed.", "category": "fail_fetch_nimbus_result", "processed": "timestamp failed at play testbed deploy nimbus timestamp task fail task path home worker workspace ansible alma linux number x nvme vmxnet number efi new gos testing internal testbed deploy deploy from database c run yml number fatal localhost failed failed to fetch nimbus testbed deploy result http sc eng vmware com qiz nimbus ansible alma linux number x nvme vmxnet number efi number testbed result testbed info json from database c servers c eng vmware com nimbus testbed deployment failed", "solution": "retry", "target": "nimbus", "version": 202206171000}, {"id": 1649, "name": "78.txt", "raw": "2022-05-26 17:55:33,026 | Failed at Play [testbed_deploy_nimbus] *********************\n2022-05-26 17:55:33,026 | TASK [fail] ************************************************\ntask path: /home/worker/workspace/Ansible_AlmaLinux_9.x_70GA_IDE_VMXNET3_BIOS/newgos_testing_internal/testbed_deploy/deploy_from_dbc_run.yml:75\nfatal: [localhost]: FAILED! => Failed to fetch nimbus testbed deploy result http://sc-dbc2154.eng.vmware.com/qiz/nimbus/Ansible_AlmaLinux_9.x_70GA_IDE_VMXNET3_BIOS-12/testbed_result/testbedInfo.json from dbc server sc-dbc2154.eng.vmware.com. Nimbus testbed deployment failed.", "category": "fail_fetch_nimbus_result", "processed": "timestamp failed at play testbed deploy nimbus timestamp task fail task path home worker workspace ansible alma linux number x number ga ide vmxnet number bios new gos testing internal testbed deploy deploy from database c run yml number fatal localhost failed failed to fetch nimbus testbed deploy result http sc eng vmware com qiz nimbus ansible alma linux number x number ga ide vmxnet number bios number testbed result testbed info json from database c servers c eng vmware com nimbus testbed deployment failed", "solution": "retry", "target": "nimbus", "version": 202206171000}, {"id": 1650, "name": "87.txt", "raw": "2022-06-01 07:32:44,001 | Failed at Play [testbed_deploy_nimbus] *********************\n2022-06-01 07:32:44,001 | TASK [fail] ************************************************\ntask path: /home/worker/workspace/Ansible_Regression_Windows_10_64/newgos_testing_internal/testbed_deploy/deploy_from_dbc_run.yml:75\nfatal: [localhost]: FAILED! => Failed to fetch nimbus testbed deploy result http://sc-dbc2154.eng.vmware.com/qiz/nimbus/Ansible_Regression_Windows_10_64-65/testbed_result/testbedInfo.json from dbc server sc-dbc2154.eng.vmware.com. Nimbus testbed deployment failed.", "category": "fail_fetch_nimbus_result", "processed": "timestamp failed at play testbed deploy nimbus timestamp task fail task path home worker workspace ansible regression windows number new gos testing internal testbed deploy deploy from database c run yml number fatal localhost failed failed to fetch nimbus testbed deploy result http sc eng vmware com qiz nimbus ansible regression windows number testbed result testbed info json from database c servers c eng vmware com nimbus testbed deployment failed", "solution": "retry", "target": "nimbus", "version": 202206171000}, {"id": 1651, "name": "93.txt", "raw": "2022-06-06 15:46:07,006 | Failed at Play [testbed_deploy_nimbus] *********************\n2022-06-06 15:46:07,006 | TASK [fail] ************************************************\ntask path: /home/worker/workspace/Ansible_Regression_Photon_4.0_OVA/newgos_testing_internal/testbed_deploy/deploy_from_dbc_run.yml:75\nfatal: [localhost]: FAILED! => Failed to fetch nimbus testbed deploy result http://sc-dbc2154.eng.vmware.com/qiz/nimbus/Ansible_Regression_Photon_4.0_OVA-73/testbed_result/testbedInfo.json from dbc server sc-dbc2154.eng.vmware.com. Nimbus testbed deployment failed.", "category": "fail_fetch_nimbus_result", "processed": "timestamp failed at play testbed deploy nimbus timestamp task fail task path home worker workspace ansible regression photon number ova new gos testing internal testbed deploy deploy from database c run yml number fatal localhost failed failed to fetch nimbus testbed deploy result http sc eng vmware com qiz nimbus ansible regression photon number ova number testbed result testbed info json from database c servers c eng vmware com nimbus testbed deployment failed", "solution": "retry", "target": "nimbus", "version": 202206171000}, {"id": 1652, "name": "95.txt", "raw": "2022-06-07 10:51:56,007 | Failed at Play [testbed_deploy_nimbus] *********************\n2022-06-07 10:51:56,007 | TASK [fail] ************************************************\ntask path: /home/worker/workspace/Ansible_Windows_11_MAIN_NVME_E1000E_EFI/newgos_testing_internal/testbed_deploy/deploy_from_dbc_run.yml:75\nfatal: [localhost]: FAILED! => Failed to fetch nimbus testbed deploy result http://sc-dbc2154.eng.vmware.com/qiz/nimbus/Ansible_Windows_11_MAIN_NVME_E1000E_EFI-43/testbed_result/testbedInfo.json from dbc server sc-dbc2154.eng.vmware.com. Nimbus testbed deployment failed.", "category": "fail_fetch_nimbus_result", "processed": "timestamp failed at play testbed deploy nimbus timestamp task fail task path home worker workspace ansible windows number main nvme efi new gos testing internal testbed deploy deploy from database c run yml number fatal localhost failed failed to fetch nimbus testbed deploy result http sc eng vmware com qiz nimbus ansible windows number main nvme efi number testbed result testbed info json from database c servers c eng vmware com nimbus testbed deployment failed", "solution": "retry", "target": "nimbus", "version": 202206171000}, {"id": 1653, "name": "23.txt", "raw": "2022-02-06 14:48:45,006 | Failed at Play [lsilogicsas_vhba_device_ops] ***************\n2022-02-06 14:48:45,006 | TASK [Execute powershell command in Windows guest] *********task path: /home/worker/workspace/Ansible_Cycle_Windows_10_32/ansible-vsphere-gos-validation/windows/utils/win_execute_cmd.yml:18\nConnectionResetError: [Errno 104] Connection reset by peer\nurllib3.exceptions.ProtocolError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))\nrequests.exceptions.ConnectionError: ('Connection aborted.', ConnectionResetError(104, ''Connection reset by peer'))\nfatal: [localhost]: FAILED! => Unexpected failure during module execution.1232123", "category": "connect_reset_unexpect_failure", "processed": "timestamp failed at play lsi logic sas v hba device ops timestamp task execute powershell command in windows guest task path home worker workspace ansible cycle windows number ansible vsphere gos validation windows utilities win execute command yml number connection reset error error number connection reset by peer url library number exceptions protocol error connection aborted connection reset error number connection reset by peer requests exceptions connection error connection aborted connection reset error number connection reset by peer fatal localhost failed unexpected failure during module execution number", "solution": "retry", "target": "targetvm", "version": 202206171000}, {"id": 1654, "name": "69.txt", "raw": "2022-05-25 06:50:25,025 | Failed at Play [wintools_complete_install_verify] **********\n2022-05-25 06:50:25,025 | TASK [Get VM 'test_vm' snapshot facts] *********************\ntask path: /home/worker/workspace/Ansible_Windows_Server_LTSC_MAIN_NVME_E1000E_BIOS/ansible-vsphere-gos-validation/common/vm_get_snapshot_facts.yml:4\nexception in /vmware.py when connect_to_api in /socket.py when create_connection\nfatal: [localhost]: FAILED! => Unknown error while connecting to vCenter or ESXi API at 10.170.69.228:443 : [Errno 110] Connection timed out", "category": "unknown_connect_vc", "processed": "timestamp failed at play win tools complete install verify timestamp task get vm test vm snapshot facts task path home worker workspace ansible windows server lts c main nvme bios ansible vsphere gos validation common vm get snapshot facts yml number exception in vmware python when connect to api in socket python when create connection fatal localhost failed unknown error while connecting to vcenter or esxi api at ip address error number connection timed out", "solution": "retry", "target": "targetvm", "version": 202206171000}, {"id": 1655, "name": "76.txt", "raw": "2022-05-26 10:18:37,026 | Failed at Play [vmlibrary_main] ****************************\n2022-05-26 10:18:37,026 | TASK [Execute powershell command 'C:\\vmlibrary\\winVMlibrary.ps1 -Config -CleanupFiles -ExeSdelete'] \ntask path: /home/worker/workspace/test_vmlibrary/ansible-vsphere-gos-validation/windows/utils/win_execute_cmd.yml:18\nfatal: [localhost -> 10.191.137.205]: FAILED! => non-zero return code when C:\\vmlibrary\\winVMlibrary.ps1\n2022-05-26 10:19:50,026 | TASK [Fail the task and quit] ******************************\ntask path: /home/worker/workspace/test_vmlibrary/newgos_testing_internal/vmlibrary/vmlibrary_main.yml:41\nfatal: [localhost]: FAILED! => Failed to export windows-11-25126-64bit as VMLibrary template from 10.191.141.87", "category": "fail_export_lib_template", "processed": "timestamp failed at play vm library main timestamp task execute powershell command c vm library win vm library ps number configuration cleanup files exes delete task path home worker workspace test vm library ansible vsphere gos validation windows utilities win execute command yml number fatal localhost ip address failed nonzero return code when c vm library win vm library ps number timestamp task fail the task and quit task path home worker workspace test vm library new gos testing internal vm library vm library main yml number fatal localhost failed failed to export windows number b it as vm library template from ip address", "solution": "deepdive", "target": "testcase", "version": 202206171000}, {"id": 1656, "name": "91.txt", "raw": "2022-06-06 02:31:46,006 | Failed at Play [deploy_vm_efi_lsilogic_vmxnet3] ************\n2022-06-06 02:31:46,006 | TASK [Datastore file operation] ****************************\ntask path: /home/worker/workspace/Ansible_Regression_Ubuntu_22.04_Server_ISO/ansible-vsphere-gos-validation/common/esxi_check_delete_datastore_file.yml:24\nfatal: [localhost]: FAILED! => File 'OS/Linux/Ubuntu/22.04/GA/ubuntu-22.04-live-server-amd64.iso' is absent, cannot continue\n2022-06-06 02:31:47,006 | TASK [Testing exit due to failure] *************************\ntask path: /home/worker/workspace/Ansible_Regression_Ubuntu_22.04_Server_ISO/ansible-vsphere-gos-validation/common/test_rescue.yml:55\nfatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_lsilogic_vmxnet3", "category": "absent_cannot_continue", "processed": "timestamp failed at play deploy vm efi lsi logic vmxnet number timestamp task data store file operation task path home worker workspace ansible regression ubuntu number server iso ansible vsphere gos validation common esxi check delete data store file yml number fatal localhost failed file os linux ubuntu number ga ubuntu number live server amd number i so is absent can not continue timestamp task testing exit due to failure task path home worker workspace ansible regression ubuntu number server iso ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi lsi logic vmxnet number", "solution": "deepdive", "target": "usererror", "version": 202206171000}, {"id": 1657, "name": "20.txt", "raw": "2022-05-23 04:57:25,023 | Failed at Play [lsilogic_vhba_device_ops] ******************\n2022-05-23 04:57:25,023 | TASK [Wait for new device to be absent] ********************\ntask path: /home/worker/workspace/Ansible_Regression_Flatcar_OVA/ansible-vsphere-gos-validation/linux/vhba_hot_add_remove/wait_device_list_changed.yml:80\nfatal: [localhost -> 10.191.154.59]: FAILED! =>  when lsblk", "category": "failed_when_lsblk", "processed": "timestamp failed at play lsi logic v hba device ops timestamp task wait for new device to be absent task path home worker workspace ansible regression flatcar ova ansible vsphere gos validation linux v hba hot add remove wait device list changed yml number fatal localhost ip address failed when ls blk", "solution": "deepdive", "target": "testcase", "version": 202206171000}, {"id": 1658, "name": "79.txt", "raw": "2022-05-28 04:24:47,028 | Failed at Play [lsilogic_vhba_device_ops] ******************\n2022-05-28 04:24:47,028 | TASK [Wait for new device to be absent] ********************\ntask path: /home/worker/workspace/Ansible_Cycle_Flatcar_LTS/ansible-vsphere-gos-validation/linux/vhba_hot_add_remove/wait_device_list_changed.yml:80\nfatal: [localhost -> 10.180.106.131]: FAILED! =>  when lsblk", "category": "failed_when_lsblk", "processed": "timestamp failed at play lsi logic v hba device ops timestamp task wait for new device to be absent task path home worker workspace ansible cycle flatcar lts ansible vsphere gos validation linux v hba hot add remove wait device list changed yml number fatal localhost ip address failed when ls blk", "solution": "deepdive", "target": "testcase", "version": 202206171000}, {"id": 1687, "name": "85.txt", "raw": "2022-06-02 07:26:17,002 | Failed at Play [testbed_deploy_nimbus] *********************\n2022-06-02 07:26:17,002 | TASK [Testbed deploy result is not PASS] *******************\ntask path: /home/worker/workspace/test_vmlibrary/newgos_testing_internal/testbed_deploy/parse_testbed_info.yml:10\nfatal: [localhost]: FAILED! => Get deployment result in testbedInfo.json is 'FAIL', not PASS", "category": "testbedinfo_fail", "processed": "timestamp failed at play testbed deploy nimbus timestamp task testbed deploy result is not pass task path home worker workspace test vm library new gos testing internal testbed deploy parse testbed info yml number fatal localhost failed get deployment result in testbed info json is fail not pass", "solution": "retry", "target": "nimbus", "version": 202206171000}, {"id": 1682, "name": "83.txt", "raw": "2022-05-30 16:49:53,030 | Failed at Play [check_quiesce_snapshot] ********************\n2022-05-30 16:49:53,030 | TASK [Revert snapshot failed] ******************************\ntask path: /home/worker/workspace/Ansible_Regression_Windows_10_64/ansible-vsphere-gos-validation/common/vm_revert_snapshot.yml:46\nfatal: [localhost]: FAILED! => Revert to snapshot 'BaseSnapshot' failed", "category": "fail_to_revert_base_snap", "processed": "timestamp failed at play check quiesce snapshot timestamp task revert snapshot failed task path home worker workspace ansible regression windows number ansible vsphere gos validation common vm revert snapshot yml number fatal localhost failed revert to snapshot base snapshot failed", "solution": "deepdive", "target": "testcase", "version": 202206171000}, {"id": 1684, "name": "15.txt", "raw": "2022-05-25 17:16:26,025 | Failed at Play [testbed_deploy_nimbus] *********************\n2022-05-25 17:16:26,025 | TASK [Testbed deploy result is not PASS] *******************\ntask path: /home/worker/workspace/Ansible_SLES_15SP4_MAIN_PARAVIRTUAL_VMXNET3_EFI/newgos_testing_internal/testbed_deploy/parse_testbed_info.yml:10\nfatal: [localhost]: FAILED! => Get deployment result in testbedInfo.json is 'FAIL', not PASS", "category": "testbedinfo_fail", "processed": "timestamp failed at play testbed deploy nimbus timestamp task testbed deploy result is not pass task path home worker workspace ansible sles main para virtual vmxnet number efi new gos testing internal testbed deploy parse testbed info yml number fatal localhost failed get deployment result in testbed info json is fail not pass", "solution": "retry", "target": "nimbus", "version": 202206171000}, {"id": 1685, "name": "16.txt", "raw": "2022-05-25 16:54:11,025 | Failed at Play [testbed_deploy_nimbus] *********************\n2022-05-25 16:54:11,025 | TASK [Testbed deploy result is not PASS] *******************\ntask path: /home/worker/workspace/Ansible_SLES_15SP4_MAIN_NVME_E1000E_BIOS/newgos_testing_internal/testbed_deploy/parse_testbed_info.yml:10\nfatal: [localhost]: FAILED! => Get deployment result in testbedInfo.json is 'FAIL', not PASS", "category": "testbedinfo_fail", "processed": "timestamp failed at play testbed deploy nimbus timestamp task testbed deploy result is not pass task path home worker workspace ansible sles main nvme bios new gos testing internal testbed deploy parse testbed info yml number fatal localhost failed get deployment result in testbed info json is fail not pass", "solution": "retry", "target": "nimbus", "version": 202206171000}, {"id": 1674, "name": "68.txt", "raw": "2022-05-25 06:58:17,025 | Failed at Play [memory_hot_add_basic] **********************\n2022-05-25 06:58:17,025 | TASK [Execute powershell command '(get-wmiobject win32_computersystem | select numberoflogicalprocessors, numberofprocessors | ft -hide | Out-String).trim()'] \ntask path: /home/worker/workspace/Ansible_Windows_Server_LTSC_70U3_LSILOGICSAS_E1000E_BIOS/ansible-vsphere-gos-validation/windows/utils/win_execute_cmd.yml:18\nsocket.timeout: The read operation timed out\nurllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='10.191.225.51', port=5986): Read timed out. (read timeout=600)\nrequests.exceptions.ReadTimeout: HTTPSConnectionPool(host='10.191.225.51', port=5986): Read timed out. (read timeout=600)\nfatal: [localhost]: FAILED! => Unexpected failure during module execution.", "category": "socket_timeout", "processed": "timestamp failed at play memory hot add basic timestamp task execute powershell command get wmi object win number computer system select number of logical processors number of processors fth ide out string trim task path home worker workspace ansible windows server lts c lsi logic sas bios ansible vsphere gos validation windows utilities win execute command yml number socket timeout the read operation timed out url library number exceptions read timeout error https connection pool host ip address port number read timed out read timeout number requests exceptions read timeout https connection pool host ip address port number read timed out read timeout number fatal localhost failed unexpected failure during module execution", "solution": "retry", "target": "targetvm", "version": 202206171000}, {"id": 1675, "name": "72.txt", "raw": "2022-05-26 03:14:26,026 | Failed at Play [lsilogicsas_vhba_device_ops] ***************\n2022-05-26 03:14:26,026 | TASK [Execute powershell command '(Get-Disk | measure).Count'] \ntask path: /home/worker/workspace/Ansible_Windows_Server_LTSC_70GA_IDE_E1000E_EFI/ansible-vsphere-gos-validation/windows/utils/win_execute_cmd.yml:18\nsocket.timeout: The read operation timed out\nurllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='10.191.154.92', port=5986): Read timed out. (read timeout=600)\nrequests.exceptions.ReadTimeout: HTTPSConnectionPool(host='10.191.154.92', port=5986): Read timed out. (read timeout=600)\nfatal: [localhost]: FAILED! => Unexpected failure during module execution.", "category": "socket_timeout", "processed": "timestamp failed at play lsi logic sas v hba device ops timestamp task execute powershell command get disk measure count task path home worker workspace ansible windows server lts c number ga ide efi ansible vsphere gos validation windows utilities win execute command yml number socket timeout the read operation timed out url library number exceptions read timeout error https connection pool host ip address port number read timed out read timeout number requests exceptions read timeout https connection pool host ip address port number read timed out read timeout number fatal localhost failed unexpected failure during module execution", "solution": "retry", "target": "targetvm", "version": 202206171000}, {"id": 1676, "name": "51.txt", "raw": "2022-05-16 08:17:27,016 | Failed at Play [gosc_perl_dhcp] ****************************\n2022-05-16 08:17:27,016 | TASK [Revert snapshot failed] ******************************\ntask path: /home/worker/workspace/Ansible_Flatcar_main/ansible-vsphere-gos-validation/common/vm_revert_snapshot.yml:46\nfatal: [localhost]: FAILED! => Revert to snapshot 'BaseSnapshot' failed", "category": "fail_to_revert_base_snap", "processed": "timestamp failed at play go sc perl dhcp timestamp task revert snapshot failed task path home worker workspace ansible flatcar main ansible vsphere gos validation common vm revert snapshot yml number fatal localhost failed revert to snapshot base snapshot failed", "solution": "deepdive", "target": "testcase", "version": 202206171000}, {"id": 1677, "name": "52.txt", "raw": "2022-05-16 08:18:01,016 | Failed at Play [gosc_cloudinit_dhcp] ***********************\n2022-05-16 08:18:01,016 | TASK [Revert snapshot failed] ******************************\ntask path: /home/worker/workspace/Ansible_Flatcar_main/ansible-vsphere-gos-validation/common/vm_revert_snapshot.yml:46\nfatal: [localhost]: FAILED! => Revert to snapshot 'BaseSnapshot' failed", "category": "fail_to_revert_base_snap", "processed": "timestamp failed at play go sc cloud init dhcp timestamp task revert snapshot failed task path home worker workspace ansible flatcar main ansible vsphere gos validation common vm revert snapshot yml number fatal localhost failed revert to snapshot base snapshot failed", "solution": "deepdive", "target": "testcase", "version": 202206171000}, {"id": 1678, "name": "53.txt", "raw": "2022-05-16 08:18:38,016 | Failed at Play [paravirtual_vhba_device_ops] ***************\n2022-05-16 08:18:38,016 | TASK [Revert snapshot failed] ******************************\ntask path: /home/worker/workspace/Ansible_Flatcar_main/ansible-vsphere-gos-validation/common/vm_revert_snapshot.yml:46\nfatal: [localhost]: FAILED! => Revert to snapshot 'BaseSnapshot' failed", "category": "fail_to_revert_base_snap", "processed": "timestamp failed at play para virtual v hba device ops timestamp task revert snapshot failed task path home worker workspace ansible flatcar main ansible vsphere gos validation common vm revert snapshot yml number fatal localhost failed revert to snapshot base snapshot failed", "solution": "deepdive", "target": "testcase", "version": 202206171000}, {"id": 1679, "name": "56.txt", "raw": "2022-05-16 08:18:57,016 | Failed at Play [lsilogicsas_vhba_device_ops] ***************\n2022-05-16 08:18:57,016 | TASK [Revert snapshot failed] ******************************\ntask path: /home/worker/workspace/Ansible_Flatcar_main/ansible-vsphere-gos-validation/common/vm_revert_snapshot.yml:46\nfatal: [localhost]: FAILED! => Revert to snapshot 'BaseSnapshot' failed", "category": "fail_to_revert_base_snap", "processed": "timestamp failed at play lsi logic sas v hba device ops timestamp task revert snapshot failed task path home worker workspace ansible flatcar main ansible vsphere gos validation common vm revert snapshot yml number fatal localhost failed revert to snapshot base snapshot failed", "solution": "deepdive", "target": "testcase", "version": 202206171000}, {"id": 1680, "name": "71.txt", "raw": "2022-05-26 03:18:39,026 | Failed at Play [vmxnet3_network_device_ops] ****************\n2022-05-26 03:18:39,026 | TASK [Revert snapshot failed] ******************************\ntask path: /home/worker/workspace/Ansible_Windows_Server_LTSC_70GA_IDE_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_revert_snapshot.yml:46\nfatal: [localhost]: FAILED! => Revert to snapshot 'BaseSnapshot' failed", "category": "fail_to_revert_base_snap", "processed": "timestamp failed at play vmxnet number network device ops timestamp task revert snapshot failed task path home worker workspace ansible windows server lts c number ga ide efi ansible vsphere gos validation common vm revert snapshot yml number fatal localhost failed revert to snapshot base snapshot failed", "solution": "deepdive", "target": "testcase", "version": 202206171000}, {"id": 1681, "name": "73.txt", "raw": "2022-05-26 03:22:52,026 | Failed at Play [memory_hot_add_basic] **********************\n2022-05-26 03:22:52,026 | TASK [Revert snapshot failed] ******************************\ntask path: /home/worker/workspace/Ansible_Windows_Server_LTSC_70GA_IDE_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_revert_snapshot.yml:46\nfatal: [localhost]: FAILED! => Revert to snapshot 'BaseSnapshot' failed", "category": "fail_to_revert_base_snap", "processed": "timestamp failed at play memory hot add basic timestamp task revert snapshot failed task path home worker workspace ansible windows server lts c number ga ide efi ansible vsphere gos validation common vm revert snapshot yml number fatal localhost failed revert to snapshot base snapshot failed", "solution": "deepdive", "target": "testcase", "version": 202206171000}, {"id": 1690, "name": "38.txt", "raw": "2022-05-16 14:44:39,016 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********\n2022-05-16 14:44:39,016 | TASK [Try to ping IP] **************************************\ntask path: /home/worker/workspace/Ansible_RHEL_8.x_MAIN_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/vm_wait_ping.yml:14\nfatal: [localhost]: FAILED! => non-zero return code when ping\n2022-05-16 14:45:11,016 | TASK [Testing exit due to failure] *************************\ntask path: /home/worker/workspace/Ansible_RHEL_8.x_MAIN_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:55\nfatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_paravirtual_vmxnet3", "category": "non_zero_when_ping_2", "processed": "timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task try to ping ip task path home worker workspace ansible rhel number x main para virtual vmxnet number efi ansible vsphere gos validation common vm wait ping yml number fatal localhost failed nonzero return code when ping timestamp task testing exit due to failure task path home worker workspace ansible rhel number x main para virtual vmxnet number efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi para virtual vmxnet number", "solution": "retry", "target": "targetvm", "version": 202206171000}, {"id": 1691, "name": "57.txt", "raw": "2022-05-17 19:27:19,017 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********\n2022-05-17 19:27:19,017 | TASK [Try to ping IP] **************************************\ntask path: /home/worker/workspace/Ansible_Cycle_Photon_4.x_Update/ansible-vsphere-gos-validation/common/vm_wait_ping.yml:14\nfatal: [localhost]: FAILED! => non-zero return code when ping\n2022-05-17 19:27:49,017 | TASK [Testing exit due to failure] *************************\ntask path: /home/worker/workspace/Ansible_Cycle_Photon_4.x_Update/ansible-vsphere-gos-validation/common/test_rescue.yml:55\nfatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_paravirtual_vmxnet3", "category": "non_zero_when_ping_2", "processed": "timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task try to ping ip task path home worker workspace ansible cycle photon number x update ansible vsphere gos validation common vm wait ping yml number fatal localhost failed nonzero return code when ping timestamp task testing exit due to failure task path home worker workspace ansible cycle photon number x update ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi para virtual vmxnet number", "solution": "retry", "target": "targetvm", "version": 202206171000}, {"id": 1694, "name": "37.txt", "raw": "2022-05-16 14:34:30,016 | Failed at Play [check_efi_firmware] ************************\n2022-05-16 14:34:30,016 | TASK [Get specified property info for VM 'test_vm'] ********\ntask path: /home/worker/workspace/Ansible_Windows_11_MAIN_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/vm_get_config.yml:4\nfatal: [localhost]: FAILED! => Unable to gather information for non-existing VM test_vm", "category": "no_info_from_not_exist_vm", "processed": "timestamp failed at play check efi firmware timestamp task get specified property info for vm test vm task path home worker workspace ansible windows number main para virtual vmxnet number efi ansible vsphere gos validation common vm get configuration yml number fatal localhost failed unable to gather information for non existing vm test vm", "solution": "deepdive", "target": "usererror", "version": 202206171000}, {"id": 1695, "name": "81.txt", "raw": "2022-05-30 07:52:59,030 | Failed at Play [check_os_fullname] *************************\n2022-05-30 07:52:59,030 | TASK [Get specified property info for VM 'test_vm'] ********\ntask path: /home/worker/workspace/Ansible_Windows_Server_LTSC_MAIN_LSILOGICSAS_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_get_config.yml:4\nfatal: [localhost]: FAILED! => Unable to gather information for non-existing VM test_vm", "category": "no_info_from_not_exist_vm", "processed": "timestamp failed at play check os full name timestamp task get specified property info for vm test vm task path home worker workspace ansible windows server lts c main lsi logic sas efi ansible vsphere gos validation common vm get configuration yml number fatal localhost failed unable to gather information for non existing vm test vm", "solution": "deepdive", "target": "usererror", "version": 202206171000}, {"id": 1697, "name": "84.txt", "raw": "2022-05-30 16:56:19,030 | Failed at Play [cpu_multicores_per_socket] *****************\n2022-05-30 16:56:19,030 | TASK [Set VM power state to 'shutdown-guest'] **************\ntask path: /home/worker/workspace/Ansible_Regression_Windows_10_64/ansible-vsphere-gos-validation/common/vm_set_power_state.yml:34\nfatal: [localhost]: FAILED! => Timeout while waiting for VM power off.", "category": "timeout_while_waiting", "processed": "timestamp failed at play cpu multi cores per socket timestamp task set vm power state to shutdown guest task path home worker workspace ansible regression windows number ansible vsphere gos validation common vm set power state yml number fatal localhost failed timeout while waiting for vm power off", "solution": "deepdive", "target": "testcase", "version": 202206171000}, {"id": 1698, "name": "30.txt", "raw": "2022-05-23 04:28:18,023 | Failed at Play [paravirtual_vhba_device_ops] ***************\n2022-05-23 04:28:18,023 | TASK [Wait for VMware Tools collecting guest info] *********\ntask path: /home/worker/workspace/Ansible_Regression_SLED_15.x/ansible-vsphere-gos-validation/common/vm_wait_guest_ip.yml:22\nfatal: [localhost]: FAILED! => hardware configuration table", "category": "origin", "processed": "timestamp failed at play para virtual v hba device ops timestamp task wait for vmware tools collecting guest info task path home worker workspace ansible regression sled number x ansible vsphere gos validation common vm wait guest ip yml number fatal localhost failed hardware configuration table", "solution": "deepdive", "target": "testcase", "version": 202206171000}, {"id": 1699, "name": "40.txt", "raw": "2022-05-23 06:16:02,023 | Failed at Play [cpu_hot_add_basic] *************************\n2022-05-23 06:16:02,023 | TASK [VM log info check failure] ***************************\ntask path: /home/worker/workspace/Ansible_Regression_Windows_Server_LTSC/ansible-vsphere-gos-validation/windows/utils/win_check_winbsod.yml:19\nfatal: [localhost]: FAILED! => Get Windows guest BSOD keyword 'WinBSOD:' in vmware.log.", "category": "origin", "processed": "timestamp failed at play cpu hot add basic timestamp task vm log info check failure task path home worker workspace ansible regression windows server lts c ansible vsphere gos validation windows utilities win check win bsod yml number fatal localhost failed get windows guest bsod keyword win bsod in vmware log", "solution": "deepdive", "target": "product", "version": 202206171000}, {"id": 1700, "name": "86.txt", "raw": "2022-06-03 02:03:55,003 | Failed at Play [ovt_verify_install] ************************\n2022-06-03 02:03:55,003 | TASK [Install packages ['open-vm-tools']] ******************\ntask path: /home/worker/workspace/Ansible_Cycle_Photon_3.x_Update/ansible-vsphere-gos-validation/linux/open_vm_tools/install_ovt.yml:35\nfatal: [localhost -> 10.180.111.126]: FAILED! => non-zero return code when tdnf\n2022-06-03 02:04:37,003 | TASK [Testing exit due to failure] *************************\ntask path: /home/worker/workspace/Ansible_Cycle_Photon_3.x_Update/ansible-vsphere-gos-validation/common/test_rescue.yml:55\nfatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case ovt_verify_install", "category": "origin", "processed": "timestamp failed at play ovt verify install timestamp task install packages open vm tools task path home worker workspace ansible cycle photon number x update ansible vsphere gos validation linux open vm tools install ovt yml number fatal localhost ip address failed nonzero return code when tdnf timestamp task testing exit due to failure task path home worker workspace ansible cycle photon number x update ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case ovt verify install", "solution": "deepdive", "target": "testcase", "version": 202206171000}, {"id": 1692, "name": "59.txt", "raw": "2022-05-23 08:04:33,023 | Failed at Play [secureboot_enable_disable] *****************\n2022-05-23 08:04:33,023 | TASK [Try to ping IP] **************************************\ntask path: /home/worker/workspace/Ansible_Regression_CentOS_8.x/ansible-vsphere-gos-validation/common/vm_wait_ping.yml:14\nfatal: [localhost]: FAILED! => non-zero return code when ping", "category": "non_zero_when_ping_2", "processed": "timestamp failed at play secure boot enable disable timestamp task try to ping ip task path home worker workspace ansible regression centos number x ansible vsphere gos validation common vm wait ping yml number fatal localhost failed nonzero return code when ping", "solution": "deepdive", "target": "targetvm", "version": 202206171000}, {"id": 1701, "name": "90.txt", "raw": "2022-06-03 02:29:30,003 | Failed at Play [deploy_vm_efi_lsilogic_vmxnet3] ************\n2022-06-03 02:29:30,003 | TASK [Wait for VMware Tools collecting guest info] *********\ntask path: /home/worker/workspace/Ansible_Regression_Photon_4.0_ISO/ansible-vsphere-gos-validation/common/vm_wait_guest_ip.yml:22\nfatal: [localhost]: FAILED! => hardware configuration table\n2022-06-03 02:29:58,003 | TASK [Testing exit due to failure] *************************\ntask path: /home/worker/workspace/Ansible_Regression_Photon_4.0_ISO/ansible-vsphere-gos-validation/common/test_rescue.yml:55\nfatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_lsilogic_vmxnet3", "category": "origin", "processed": "timestamp failed at play deploy vm efi lsi logic vmxnet number timestamp task wait for vmware tools collecting guest info task path home worker workspace ansible regression photon number iso ansible vsphere gos validation common vm wait guest ip yml number fatal localhost failed hardware configuration table timestamp task testing exit due to failure task path home worker workspace ansible regression photon number iso ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi lsi logic vmxnet number", "solution": "retry", "target": "targetvm", "version": 202206171000}, {"id": 1702, "name": "29.txt", "raw": "2022-05-23 04:00:26,023 | Failed at Play [lsilogic_vhba_device_ops] ******************\n2022-05-23 04:00:26,023 | TASK [Wait for new device to be present] *******************\ntask path: /home/worker/workspace/Ansible_Regression_Ubuntu_22.04_Server_OVA/ansible-vsphere-gos-validation/linux/vhba_hot_add_remove/wait_device_list_changed.yml:80\nfatal: [localhost]: UNREACHABLE! => Failed to create temporary directory.In some cases, you may have been able to authenticate and did not have permissions on the target directory. Consider changing the remote tmp path in ansible.cfg to a path rooted in \"/tmp\", for more error information use -vvv. Failed command was: ( umask 77 && mkdir -p \"` echo /tmp `\"&& mkdir \"` echo /tmp/ansible-tmp-1653278426.1675-6193-215757058899332 `\" && echo ansible-tmp-1653278426.1675-6193-215757058899332=\"` echo /tmp/ansible-tmp-1653278426.1675-6193-215757058899332 `\" ), exited with result 1\n2022-05-23 04:00:26,023 | TASK [Guest OS unreachable] ********************************\ntask path: /home/worker/workspace/Ansible_Regression_Ubuntu_22.04_Server_OVA/ansible-vsphere-gos-validation/linux/vhba_hot_add_remove/wait_device_list_changed.yml:91\nfatal: [localhost]: FAILED! => {'changed': False, 'msg': 'Failed to create temporary directory.In some cases, you may have been able to authenticate and did not have permissions on the target directory. Consider changing the remote tmp path in ansible.cfg to a path rooted in \"/tmp\", for more error information use -vvv. Failed command was: ( umask 77 && mkdir -p \"` echo /tmp `\"&& mkdir \"` echo /tmp/ansible-tmp-1653278426.1675-6193-215757058899332 `\" && echo ansible-tmp-1653278426.1675-6193-215757058899332=\"` echo /tmp/ansible-tmp-1653278426.1675-6193-215757058899332 `\" ), exited with result 1', 'skip_reason': 'Host localhost is unreachable', 'unreachable': True}", "category": "fail_create_temporary_dir_auth", "processed": "timestamp failed at play lsi logic v hba device ops timestamp task wait for new device to be present task path home worker workspace ansible regression ubuntu number server ova ansible vsphere gos validation linux v hba hot add remove wait device list changed yml number fatal localhost un reachable failed to create temporary directory in some cases you may have been able to authenticate and did not have permissions on the target directory consider changing the remote tmp path in ansible configuration to a path rooted in tmp for more error information use v v v failed command was u mask number mkdir p echo tmp mkdir echo tmp ansible tmp hex id timestamp echo ansible tmp hex id timestamp echo tmp ansible tmp hex id timestamp exited with result number timestamp task guest os un reachable task path home worker workspace ansible regression ubuntu number server ova ansible vsphere gos validation linux v hba hot add remove wait device list changed yml number fatal localhost failed changed false message failed to create temporary directory in some cases you may have been able to authenticate and did not have permissions on the target directory consider changing the remote tmp path in ansible configuration to a path rooted in tmp for more error information use v v v failed command was u mask number mkdir p echo tmp mkdir echo tmp ansible tmp hex id timestamp echo ansible tmp hex id timestamp echo tmp ansible tmp hex id timestamp exited with result number skip reason host localhost is un reachable un reachable true", "solution": "deepdive", "target": "testcase", "version": 202206171000}, {"id": 1703, "name": "32.txt", "raw": "2022-05-16 04:08:01,016 | Failed at Play [gosc_sanity_dhcp] **************************\n2022-05-16 04:08:01,016 | TASK [Get the absolute path in Windows guest] **************\ntask path: /home/worker/workspace/Ansible_Windows_11_67U3_LSILOGICSAS_E1000E_EFI/ansible-vsphere-gos-validation/windows/utils/win_get_path.yml:10\nfatal: [localhost]: UNREACHABLE! => Failed to connect to the host via PSRP: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))", "category": "unreachable_host", "processed": "timestamp failed at play go sc sanity dhcp timestamp task get the absolute path in windows guest task path home worker workspace ansible windows number lsi logic sas efi ansible vsphere gos validation windows utilities win get path yml number fatal localhost un reachable failed to connect to the host via psrp connection aborted connection reset error number connection reset by peer", "solution": "retry", "target": "targetvm", "version": 202206171000}, {"id": 1704, "name": "96.txt", "raw": "2022-06-10 06:54:11,010 | Failed at Play [gosc_sanity_dhcp] **************************\n2022-06-10 06:54:11,010 | TASK [Shutdown guest OS inside OS] *************************\ntask path: /home/worker/workspace/Ansible_Windows_11_70GA_IDE_E1000E_EFI/ansible-vsphere-gos-validation/windows/utils/win_shutdown_restart.yml:8\nfatal: [localhost]: UNREACHABLE! => Failed to connect to the host via PSRP: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))", "category": "unreachable_host", "processed": "timestamp failed at play go sc sanity dhcp timestamp task shutdown guest os inside os task path home worker workspace ansible windows number ga ide efi ansible vsphere gos validation windows utilities win shutdown restart yml number fatal localhost un reachable failed to connect to the host via psrp connection aborted connection reset error number connection reset by peer", "solution": "retry", "target": "targetvm", "version": 202206171000}, {"id": 1705, "name": "49.txt", "raw": "2022-05-23 03:57:01,023 | Failed at Play [deploy_vm_efi_nvme_e1000e] *****************\n2022-05-23 03:57:01,023 | TASK [Check VM 'test_windows11_22538' IP address] **********\ntask path: /home/worker/workspace/Ansible_Regression_Windows_11_64/ansible-vsphere-gos-validation/common/vm_get_ip.yml:36\nfatal: [localhost]: FAILED! => Failed to get VM 'test_windows11_22538' IP Address\n2022-05-23 03:58:21,023 | TASK [Testing exit due to failure] *************************\ntask path: /home/worker/workspace/Ansible_Regression_Windows_11_64/ansible-vsphere-gos-validation/common/test_rescue.yml:55\nfatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_nvme_e1000e", "category": "fail_to_get_vm", "processed": "timestamp failed at play deploy vm efi nvme timestamp task check vm test windows number ip address task path home worker workspace ansible regression windows number ansible vsphere gos validation common vm get ip yml number fatal localhost failed failed to get vm test windows number ip address timestamp task testing exit due to failure task path home worker workspace ansible regression windows number ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi nvme e number e", "solution": "retry", "target": "targetvm", "version": 202206171000}, {"id": 1706, "name": "75.txt", "raw": "2022-05-26 05:02:19,026 | Failed at Play [deploy_vm_efi_paravirtual_e1000e] **********\n2022-05-26 05:02:19,026 | TASK [Check VM 'test_vm' IP address] ***********************\ntask path: /home/worker/workspace/Ansible_Cycle_Windows_11_64/ansible-vsphere-gos-validation/common/vm_get_ip.yml:36\nfatal: [localhost]: FAILED! => Failed to get VM 'test_vm' IP Address\n2022-05-26 05:02:56,026 | TASK [Testing exit due to failure] *************************\ntask path: /home/worker/workspace/Ansible_Cycle_Windows_11_64/ansible-vsphere-gos-validation/common/test_rescue.yml:55\nfatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_paravirtual_e1000e", "category": "fail_to_get_vm", "processed": "timestamp failed at play deploy vm efi para virtual timestamp task check vm test vm ip address task path home worker workspace ansible cycle windows number ansible vsphere gos validation common vm get ip yml number fatal localhost failed failed to get vm test vm ip address timestamp task testing exit due to failure task path home worker workspace ansible cycle windows number ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi para virtual e number e", "solution": "retry", "target": "targetvm", "version": 202206171000}, {"id": 1707, "name": "94.txt", "raw": "2022-06-07 02:17:42,007 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********\n2022-06-07 02:17:42,007 | TASK [Check VM 'test_vm' IP address] ***********************\ntask path: /home/worker/workspace/test_vmlibrary/ansible-vsphere-gos-validation/common/vm_get_ip.yml:36\nfatal: [localhost]: FAILED! => Failed to get VM 'test_vm' IP Address\n2022-06-07 02:18:16,007 | TASK [Testing exit due to failure] *************************\ntask path: /home/worker/workspace/test_vmlibrary/ansible-vsphere-gos-validation/common/test_rescue.yml:55\nfatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_paravirtual_vmxnet3", "category": "fail_to_get_vm", "processed": "timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task check vm test vm ip address task path home worker workspace test vm library ansible vsphere gos validation common vm get ip yml number fatal localhost failed failed to get vm test vm ip address timestamp task testing exit due to failure task path home worker workspace test vm library ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi para virtual vmxnet number", "solution": "retry", "target": "targetvm", "version": 202206171000}, {"id": 1708, "name": "25.txt", "raw": "2022-05-16 11:19:43,016 | Failed at Play [testbed_deploy_nimbus] *********************\n2022-05-16 11:19:43,016 | TASK [Parameter error] *************************************\ntask path: /home/worker/workspace/Ansible_Cycle_Photon_3.x_Update/newgos_testing_internal/testbed_deploy/testbed_deploy_nimbus.yml:60\nfatal: [localhost]: FAILED! => Please specify vCenter build number in command line or in vars/internal_test.yml using 'vcenter_build_num' parameter", "category": "no_vcenter_number", "processed": "timestamp failed at play testbed deploy nimbus timestamp task parameter error task path home worker workspace ansible cycle photon number x update new gos testing internal testbed deploy testbed deploy nimbus yml number fatal localhost failed please specify vcenter build number in command line or in vars internal test yml using vcenter build number parameter", "solution": "deepdive", "target": "usererror", "version": 202206171000}, {"id": 1709, "name": "60.txt", "raw": "2022-05-17 18:47:02,017 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********\n2022-05-17 18:47:02,017 | TASK [Wait for port 22 to become open or contain specific keyword] \ntask path: /home/worker/workspace/Ansible_Cycle_CentOS_8.x/ansible-vsphere-gos-validation/common/vm_wait_ssh.yml:19\nfatal: [localhost]: FAILED! => Timeout when waiting for search string OpenSSH in 10.191.184.117:22\n2022-05-17 18:47:39,017 | TASK [Testing exit due to failure] *************************\ntask path: /home/worker/workspace/Ansible_Cycle_CentOS_8.x/ansible-vsphere-gos-validation/common/test_rescue.yml:55\nfatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_paravirtual_vmxnet3", "category": "timeout_waiting_search_openssh", "processed": "timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task wait for port number to become open or contain specific keyword task path home worker workspace ansible cycle centos number x ansible vsphere gos validation common vm wait ssh yml number fatal localhost failed timeout when waiting for search string openssh in ip address timestamp task testing exit due to failure task path home worker workspace ansible cycle centos number x ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi para virtual vmxnet number", "solution": "retry", "target": "targetvm", "version": 202206171000}, {"id": 1710, "name": "100.txt", "raw": "2022-06-10 05:51:17,010 | Failed at Play [deploy_vm_efi_sata_vmxnet3] ****************\n2022-06-10 05:51:17,010 | TASK [Datastore file operation] ****************************\ntask path: /home/worker/workspace/dw_RHEL9_ARM/ansible-vsphere-gos-validation/common/esxi_check_delete_datastore_file.yml:24\nfatal: [localhost]: FAILED! => Failed to query for file 'ISO/rhel-baseos-9.0-aarch64-boot.iso'\n2022-06-10 05:51:18,010 | TASK [Testing exit due to failure] *************************\ntask path: /home/worker/workspace/dw_RHEL9_ARM/ansible-vsphere-gos-validation/common/test_rescue.yml:55\nfatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_sata_vmxnet3", "category": "manual", "processed": "timestamp failed at play deploy vm efi sata vmxnet number timestamp task data store file operation task path home worker workspace dw rhel number arm ansible vsphere gos validation common esxi check delete data store file yml number fatal localhost failed failed to query for file iso rhel base os number a arch number boot iso timestamp task testing exit due to failure task path home worker workspace dw rhel number arm ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi sata vmxnet number", "solution": "deepdive", "target": "testcase", "version": 202206171000}, {"id": 1711, "name": "101.txt", "raw": "2022-06-08 07:21:26,008 | Failed at Play [deploy_vm_efi_sata_vmxnet3] ****************\n2022-06-08 07:21:26,008 | TASK [Set VM power state to 'powered-on'] ******************\ntask path: /home/worker/workspace/dw_RHEL9_ARM/ansible-vsphere-gos-validation/common/vm_set_power_state.yml:34\nfatal: [localhost]: FAILED! => The serial port output file \"serial-20220608072106.log\" already exists. Do you want to replace it with new content or append new content to the end of the file?\n2022-06-08 07:21:51,008 | TASK [Testing exit due to failure] *************************\ntask path: /home/worker/workspace/dw_RHEL9_ARM/ansible-vsphere-gos-validation/common/test_rescue.yml:55\nfatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_sata_vmxnet3", "category": "manual", "processed": "timestamp failed at play deploy vm efi sata vmxnet number timestamp task set vm power state to powered on task path home worker workspace dw rhel number arm ansible vsphere gos validation common vm set power state yml number fatal localhost failed the serial port output file serial timestamp log already exists do you want to replace it with new content or append new content to the end of the file timestamp task testing exit due to failure task path home worker workspace dw rhel number arm ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi sata vmxnet number", "solution": "deepdive", "target": "testcase", "version": 202206171000}, {"id": 1622, "name": "66.txt", "raw": "2022-05-24 09:04:09,024 | Failed at Play [gosc_perl_staticip] ************************\n2022-05-24 09:04:09,024 | TASK [Fetch file /tmp/resolv.conf from VM guest] ***********\ntask path: /home/worker/workspace/Ansible_Cycle_Debian_11.x_32bit/ansible-vsphere-gos-validation/common/vm_guest_file_operation.yml:93\nexception in /vmware_guest_file_operation.py when fetch in /SoapAdapter.py when InvokeMethod\nfatal: [localhost]: FAILED! => Failed to Fetch file from Vm VMware exception : (vim.fault.GuestOperationsUnavailable) {\n   dynamicType = <unset>,\n   dynamicProperty = (vmodl.DynamicProperty) [],\n   msg = 'The guest operations agent could not be contacted.',\n   faultCause = <unset>,\n   faultMessage = (vmodl.LocalizableMessage) []\n}", "category": "guestos_not_contacted", "processed": "timestamp failed at play go sc perl static ip timestamp task fetch file tmp resolve configuration from vm guest task path home worker workspace ansible cycle debian number x number b it ansible vsphere gos validation common vm guest file operation yml number exception in vmware guest file operation python when fetch in soap adapter python when invoke method fatal localhost failed failed to fetch file from vm vmware exception vim fault guest operations unavailable dynamic type un set dynamic property vmodl dynamic property message the guest operations agent could not be contacted fault cause un set fault message vmodl localizable message ", "solution": "retry", "target": "targetvm", "version": 202206171000}, {"id": 1623, "name": "97.txt", "raw": "2022-06-10 07:38:49,010 | Failed at Play [gosc_sanity_staticip] **********************\n2022-06-10 07:38:49,010 | TASK [Customize Windows guest OS] **************************\ntask path: /home/worker/workspace/Ansible_Windows_11_70U1_SATA_E1000E_EFI/ansible-vsphere-gos-validation/windows/guest_customization/win_gosc_execution.yml:4\nfatal: [localhost]: FAILED! => Customization failed. For detailed information see warnings Waiting for customization result event timed out.\n2022-06-10 07:39:35,010 | TASK [Fetch file C:\\Windows\\Temp\\vmware-imc\\guestcust.log from VM guest] \ntask path: /home/worker/workspace/Ansible_Windows_11_70U1_SATA_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_guest_file_operation.yml:93\nexception in /vmware_guest_file_operation.py when fetch in /SoapAdapter.py when InvokeMethod\nfatal: [localhost]: FAILED! => Failed to Fetch file from Vm VMware exception : (vim.fault.GuestOperationsUnavailable) {\n   dynamicType = <unset>,\n   dynamicProperty = (vmodl.DynamicProperty) [],\n   msg = 'The guest operations agent could not be contacted.',\n   faultCause = <unset>,\n   faultMessage = (vmodl.LocalizableMessage) []\n}", "category": "guestos_not_contacted", "processed": "timestamp failed at play go sc sanity static ip timestamp task customize windows guest os task path home worker workspace ansible windows number sata efi ansible vsphere gos validation windows guest customization win go sc execution yml number fatal localhost failed customization failed for detailed information see warnings waiting for customization result event timed out timestamp task fetch file c windows temp vmware imc guest cust log from vm guest task path home worker workspace ansible windows number sata efi ansible vsphere gos validation common vm guest file operation yml number exception in vmware guest file operation python when fetch in soap adapter python when invoke method fatal localhost failed failed to fetch file from vm vmware exception vim fault guest operations unavailable dynamic type un set dynamic property vmodl dynamic property message the guest operations agent could not be contacted fault cause un set fault message vmodl localizable message ", "solution": "deepdive", "target": "testcase", "version": 202206171000}, {"id": 1696, "name": "82.txt", "raw": "2022-05-30 16:46:51,030 | Failed at Play [memory_hot_add_basic] **********************\n2022-05-30 16:46:51,030 | TASK [Set VM power state to 'shutdown-guest'] **************\ntask path: /home/worker/workspace/Ansible_Regression_Windows_10_64/ansible-vsphere-gos-validation/common/vm_set_power_state.yml:34\nfatal: [localhost]: FAILED! => Timeout while waiting for VM power off.", "category": "timeout_while_waiting", "processed": "timestamp failed at play memory hot add basic timestamp task set vm power state to shutdown guest task path home worker workspace ansible regression windows number ansible vsphere gos validation common vm set power state yml number fatal localhost failed timeout while waiting for vm power off", "solution": "deepdive", "target": "targetvm", "version": 202206171000}, {"id": 1688, "name": "31.txt", "raw": "2022-05-23 08:49:41,023 | Failed at Play [secureboot_enable_disable] *****************\n2022-05-23 08:49:41,023 | TASK [Try to ping IP] **************************************\ntask path: /home/worker/workspace/Ansible_Regression_RHEL_7.x/ansible-vsphere-gos-validation/common/vm_wait_ping.yml:14\nfatal: [localhost]: FAILED! => non-zero return code when ping", "category": "non_zero_when_ping_2", "processed": "timestamp failed at play secure boot enable disable timestamp task try to ping ip task path home worker workspace ansible regression rhel number x ansible vsphere gos validation common vm wait ping yml number fatal localhost failed nonzero return code when ping", "solution": "deepdive", "target": "targetvm", "version": 202206171000}, {"id": 1689, "name": "36.txt", "raw": "2022-05-16 11:50:01,016 | Failed at Play [secureboot_enable_disable] *****************\n2022-05-16 11:50:01,016 | TASK [Try to ping IP] **************************************\ntask path: /home/worker/workspace/Ansible_Cycle_Photon_3.x_Update/ansible-vsphere-gos-validation/common/vm_wait_ping.yml:14\nfatal: [localhost]: FAILED! => non-zero return code when ping", "category": "non_zero_when_ping_2", "processed": "timestamp failed at play secure boot enable disable timestamp task try to ping ip task path home worker workspace ansible cycle photon number x update ansible vsphere gos validation common vm wait ping yml number fatal localhost failed nonzero return code when ping", "solution": "deepdive", "target": "targetvm", "version": 202206171000}, {"id": 1693, "name": "88.txt", "raw": "2022-06-03 05:14:47,003 | Failed at Play [secureboot_enable_disable] *****************\n2022-06-03 05:14:47,003 | TASK [Try to ping IP] **************************************\ntask path: /home/worker/workspace/Ansible_Cycle_RHEL_7.x/ansible-vsphere-gos-validation/common/vm_wait_ping.yml:14\nfatal: [localhost]: FAILED! => non-zero return code when ping", "category": "non_zero_when_ping_2", "processed": "timestamp failed at play secure boot enable disable timestamp task try to ping ip task path home worker workspace ansible cycle rhel number x ansible vsphere gos validation common vm wait ping yml number fatal localhost failed nonzero return code when ping", "solution": "deepdive", "target": "targetvm", "version": 202206171000}]